android-sdk/android/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -111,7 +133,6 @@ open class LogtoClient(
                     completion?.onComplete(it)
                     return@getOidcConfig
                 }
```

Comment:
nit: Since signInWithBrowser is a big method, making it returns an inline `getOidcConfig` returning method is not clear enough. Prefer a more traditional way. like 

```
fun signInWithBrowser{
 val oidcConfig = getOidcConfig():
 ...
}
```

That would be more readable.

Reply:
_**1. `return@getOidcConfig`**_
this is the way kotlin works.
<img width="566" alt="image" src="https://user-images.githubusercontent.com/10806653/153786995-89c1eb74-9b89-47f9-b1f5-49a12dc9fcce.png">

**_2. Replace `getOidcConfig` with a synchronized method._**
This method will ask the backend for results. Unless we make the request process become a blocking request, it's impossible to make this method synchronized.